manifest.json

```
@@ -1,7 +1,7 @@
 {
   "name": "google-tag-manager",
   "vendor": "vtex",
-  "version": "3.3.1",
+  "version": "3.3.2",
```

Our bot updates the version

```suggestion
  "version": "3.3.1",
```